pkg/apis/garden/types.go

```diff
@@ -466,17 +466,28 @@ type SeedSpec struct {
 	Networks SeedNetworks
 	// BlockCIDRs is a list of network addresses tha should be blocked for shoot control plane components running
 	// in the seed cluster.
-	BlockCIDRs []gardencore.CIDR
+	BlockCIDRs []CIDR
 	// Visible labels the Seed cluster as selectable for the seedfinder admission controller.
 	Visible *bool
 	// Protected prevent that the Seed Cluster can be used for regular Shoot cluster control planes.
 	Protected *bool
+	// Volume contains settings for persistentvolumes created in the seed cluster.
+	Volume *SeedVolume
 }
 
+const (
+	MigrationSeedCloudProfile      = "seed.migration.gardener.cloud/cloudProfile"
```

Comment:
I'd rather go for `migration.seed` as you're about to remove the '`migration` namespace' completely.